content/docs/components/kbd/theming.mdx

```diff
@@ -2,3 +2,181 @@
 id: kbd
 scope: theming
 ---
+
+The `Kbd` component is a single part component. All of the styling is applied
+directly to the `kbd` element.
```

Comment:
Capital for the Kbd element.
```suggestion
directly to the `Kbd` element.
```

Reply:
hmmm, here I mean <kbd> html element. Do you agree with me to change it to <kbd>?

Reply:
Ah I see. No need for a change.